src/script/engines/lua.c

```diff
@@ -390,6 +451,8 @@ struct mScriptEngineContext* _luaCreate(struct mScriptEngine2* engine, struct mS
 
 	luaL_newmetatable(luaContext->lua, "mSTStruct");
 #if LUA_VERSION_NUM < 502
+	lua_pushliteral(luaContext->lua, "mSTStruct");
```

Comment:
I become tempted to refactor these to reduce just how much repetition is involved here, but that's kinda out of scope for this PR.

Reply:
Yes, I'm definitely missing something here. I agree with this needing refactorization but I don't know how at the momment.

Reply:
(I guess that should probably be declared `static`.)